lib/src/stereoCamera.cpp

```diff
@@ -105,74 +153,8 @@ StereoCamera::StereoCamera(Camera Left, Camera Right,bool rectify) {
     cv::initModule_nonfree();
 #endif 
 
-    use_elas = false;
 }
 
-void StereoCamera::initELAS(yarp::os::ResourceFinder &rf)
```

Comment:
This was a bad move @damianomal as SFM still needs this.
I'm gonna reinstate it; don't know the implications it'll have on `DispModule` though.